packages/app-project/pages/_app.js

```diff
@@ -1,13 +1,14 @@
 import zooTheme from '@zooniverse/grommet-theme'
 import { ZooFooter } from '@zooniverse/react-components'
-import { Grommet } from 'grommet'
+import { Grommet, base } from 'grommet'
 import makeInspectable from 'mobx-devtools-mst'
 import { Provider } from 'mobx-react'
 import { getSnapshot } from 'mobx-state-tree'
 import App, { Container } from 'next/app'
 import React from 'react'
 import { createGlobalStyle } from 'styled-components'
 import UrlParse from 'url-parse'
+import _ from 'lodash'
```

Comment:
We could use a method import here to enable tree-shaking:

```
import merge from 'lodash/merge`
```

(This probably needs to be done elsewhere too)

Reply:
oh, good point, thanks.